client/benches/tps/README.md

```diff
@@ -0,0 +1,35 @@
+# tps
+
+## Usage
+
+1. Establish a baseline:
+
+    ```
+    git checkout iroha2-dev
+    ```
+    ```
+    cargo bench --bench tps-dev
+    ```
+
+2. Compare against the baseline:
+
+    ```
+    git checkout <your-optimization-commit>
+    ```
+    ```
+    cargo bench --bench tps-dev
+    ```
+    * CAUTION: "improved" and "regressed" message are reversed due to the nature of Criterion.rs, which is supposed to measure time
```

Comment:
I can't understand why... At step 1 criterion saves its results into `target` directory. At step 2 it compares new results with previous. Why does Criterion swap this words?

Reply:
Because in TPS time is the denominator